mender-deb-package

```diff
@@ -72,7 +72,7 @@ cat << EOF > debian/rules
 	dh \$@
 
 override_dh_auto_build:
-	make CGO_CFLAGS="-I${LIBLZMA_INSTALL_PATH}/include" CGO_LDFLAGS="-L${LIBLZMA_INSTALL_PATH}/lib" \
+	make CGO_CFLAGS="-idirafter/usr/include" CGO_LDFLAGS="-L/usr/lib/arm-linux-gnueabihf/" \
```

Comment:
I went for the compiler flag solution. However, for now `arm-linux-gnueabihf` is hardcoded. Does it need to be variabelized?